pay-api/src/pay_api/services/bcol_service.py

```diff
@@ -49,6 +49,7 @@ def get_payment_system_code(self):
     def create_invoice(self, payment_account: PaymentAccount,  # pylint: disable=too-many-locals
                        line_items: [PaymentLineItem], invoice: Invoice, **kwargs) -> InvoiceReference:
         """Create Invoice in PayBC."""
+        from pay_api.services import flags  # pylint: disable=import-outside-toplevel
```

Comment:
If I didn't put the import here, ran into circular dependency troubles or flags not having `is_on`